tests/test_operations/test_transform.py

```diff
@@ -22,6 +22,8 @@ def test_spherize():
     spherize_methods = ["PCA", "ZCA", "PCA-cor", "ZCA-cor"]
     for method in spherize_methods:
         for center in [True, False]:
+            if ["PCA-cor", "ZCA-cor"] and not center:
+                continue
```

Comment:
Can you add a test that enters the assertion? This is a great spot for it!

Reply:
I do this in `test_normalize_spherize` instead